### PR TITLE
Electron-90 (Notification Position)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
Spectron test for notification position

### CODE REVIEW

File: spectron/tests/notificationPosition-test.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    it('should check notification position', () => {
+        return app.browserWindow
+            .getBounds().then((bounds) => {
+                bounds.x.should.be.equal(0);
+                bounds.y.should.be.above(0);
+            });
+    });
+
+    it('should change the window', () => {
```
Comment (VikasShashidhar) [inline spectron/tests/notificationPosition-test.js]: Do we need to write these as a test? We can use them as general logical statements.

File: spectron/tests/notificationPosition-test.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    it('should check notification position and equal to lower-right', () => {
+        return app.browserWindow
+            .getBounds().then((bounds) => {
+                bounds.x.should.be.above(0);
+                bounds.y.should.be.above(0);
+            });
+    });
+
+    it('should change the window', () => {
+        return app.client.windowByIndex(0);
```
Comment (VikasShashidhar) [inline spectron/tests/notificationPosition-test.js]: Same as above.

File: spectron/tests/notificationPosition-test.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    it('should check notification position and equal to upper-right', () => {
+        return app.browserWindow
+            .getBounds().then((bounds) => {
+                bounds.x.should.be.above(0);
+                bounds.y.should.be.above(0);
+            });
+    });
+
+    it('should close the notification', () => {
+        return app.client
```
Comment (VikasShashidhar) [inline spectron/tests/notificationPosition-test.js]: Same as above.

**Comment by KiranNiranjan** [comment]
@VikasShashidhar Please review. Thanks

File: tests/notificationPosition.test.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    });
+
+    it('should load demo html', async () => {
+        return app.client.waitUntilWindowLoaded().then(async () => {
+            const title = await app.client.getTitle();
+            expect(title === '').toBeTruthy();
+        });
+    });
+
+    it('should notification configure window', () => {
```
Comment (VikasShashidhar) [inline tests/notificationPosition.test.js]: This should be "should open notification configure window"

**Comment by VikasShashidhar** [comment]
LGTM 👍

**Pull request merged**